### DISCUSSION TITLE
Conditional plugin configuration

### DISCUSSION THREAD

**Discussion by stinovlas** [python-lsp/python-lsp-server#351] [Configuration]
Hi! I recently migrated to `python-lsp-server` from `jedi-language-server`. I figure whether there is some way to conditionaly enable/disable plugins for some projects. I do have a number of them and some of them use some plugins, others don't (e.g. some of the projects I work on are formatted by `black` and some are not).

What I'm asking for is not configuration of the plugin, but rather configuration of `python-lsp-server` whether to use that plugin for a particular project. Is that even possible?

Edit: I'll probably be able to create some hooks for my neovim config to set whether plugin is enabled, if necessary, but I was wondering whether there is some way how to configure `python-lsp-server` directly.

**Comment by ccordoba12** [comment]
Hey @stinovlas, thanks for reporting. I don't think this is is possible, i.e. `pylsp` doesn't support per project configurations.
